src/HDF5.jl

```diff
@@ -931,7 +970,18 @@ end
 t_commit(parent::Union{HDF5File, HDF5Group}, path::String, dtype::HDF5Datatype) = t_commit(parent, path, dtype, p_create(H5P_LINK_CREATE))
 
 a_create(parent::Union{HDF5File, HDF5Object}, name::String, dtype::HDF5Datatype, dspace::HDF5Dataspace) = HDF5Attribute(h5a_create(checkvalid(parent).id, name, dtype.id, dspace.id), file(parent))
-p_create(class, toclose=false) = HDF5Properties(h5p_create(class), toclose)
+p_create(class, toclose=false) = HDF5Properties(h5p_create(class), toclose, class)
+function p_create(class, toclose, prop1::String, val1, pv...) 
+    p=p_create(class, toclose)
+    t = (prop1, val1, pv...)
+    checkprops(t...)
+    for i=1:2:length(t)
+        if hdf5_prop_get_set[t[i]][3] == class
```

Comment:
does this silently ignore non-applicable properties? should it error instead?

Reply:
Yes, non-applicable props are ignored. The idea was to have a single list for all properties, throuhout the call tree. I don't know if it's a good idea to raise errors, since also default (empty) properties are created using this function. At the moment errors will only be raised, if the property key does not exist in the dict.